tree/ntuple/v7/inc/ROOT/RNTupleDescriptor.hxx

```diff
@@ -392,7 +392,37 @@ writte struct. This allows for forward and backward compatibility when the meta-
 class RNTupleDescriptor {
    friend class RNTupleDescriptorBuilder;
 
+public:
+   class RFieldDescriptorIterable;
+
 private:
+   class RHeaderExtension {
```

Comment:
Since we have a public `GetHeaderExtension()`, shouldn't the class definition be public as well?